### ISSUE TITLE
provide and use a global namespace

### ISSUE THREAD

**Issue by thenewguy** [chrivers/pyjaco#66]
Output should be namespaced and **builtins** should be accessed from this namespace.  I would guess a default of pyjaco would be used, but it should also be configurable.

**Comment by chrivers** [comment]
Yes, we are working to clean up the namespace situation, while fixing imports. "pyjaco" is probably a good bet for the default namespace.

**Comment by thenewguy** [comment]
I think we should actually use "π" as the default javascript namespace.  This could be configurable in case pyjaco is building for an environment that does not support unicode identifiers?
